src/pages/Getstarted/Getstarted.tsx

```diff
@@ -0,0 +1,37 @@
+import { FC } from 'react'
+import Logo from './logo192.png'
+
+const Getstarted: FC = () => (
+  <div className="flex">
```

Comment:
there should have a <Container> to contrl the maxwith is 1280P.
```suggestion
<Container>
  <div className="flex">
```